### ISSUE TITLE
[import/order] Feature request: ignore non top level imports

### ISSUE THREAD

**Issue by afrokick** [benmosher/eslint-plugin-import#1750]
An example:

Options:
```
"alphabetize": {
  "order": "asc"
}
```

```javascript
import { B } from 'module-b'

export function methodA() {
  if (SOME_ENV) {
    import { A } from 'module-a'

    return A.methodA()
  }
}
```

> ESLint: `medule-a` import should occur before import of `module-b`(import/order)

The rule should applies only with imports from the same level(scope).

**Comment by ljharb** [comment]
That's entirely invalid syntax; import statements can only ever appear at the top level of a module.
